app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,303 @@
+---
+title: MeshCircuitBreaker (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state, it should not be mixed
+with [CircuitBreaker](../circuit-breaker).
+{% endwarning %}
+
+This policy will look for errors in the live traffic being exchanged between our data plane proxies and it will mark a
+data proxy as an unhealthy if certain conditions are met and - by doing so - making sure that no additional traffic can
+reach an unhealthy data plane proxy until it is healthy again.
```

Comment:
how about shorter sentences? Something like
```
This policy will look for errors in the live traffic being exchanged between our data plane proxies.
It will mark a data proxy as unhealthy if certain conditions are met.
The policy will ensure that no additional traffic can reach an unhealthy data plane proxy until it is healthy again.
```

Reply:
addressed